src/app.ts

```diff
@@ -0,0 +1,22 @@
+/// <reference path='../typings/tsd.d.ts' />
```

Comment:
the idea is to provide an angular module, that can maybe also contain some handy services to load, group or parse data